src/js/_enqueues/wp/updates.js

```diff
@@ -1454,14 +1458,14 @@
 				$itemsCount.text(
 					sprintf(
 						/* translators: %s: The remaining number of plugins. */
-						_nx( '%s item', '%s items', 'plugin/plugins', remainingCount ),
+						_nx( '%s item', '%s items', remainingCount, 'plugin/plugins'  ),
```

Comment:
Can't wait for [ESLint integration](https://core.trac.wordpress.org/ticket/31823) so this will get caught automatically